website/docs/remotedeploydir.md

```diff
@@ -0,0 +1,23 @@
+Directory on the remote machine where the project will be deployed to.
+
+```lua
+remoteprojectdir ("path")
+```
+
+### Parameters ###
+
+`path` specifies the directory on the remote machine where the project is deployed
+
+### Applies To ###
+
+The `config` scope.
+
+### Availability ###
+
+Premake 5.0.0 alpha 14 or later, only applies to Visual Studio Linux projects.
```

Comment:
This availability is wrong. It would be the next beta version or later.

Reply:
Fixed via b16716229a1312b82434add4b53dfe6e66fc88ae